### ISSUE TITLE
Stop commands in parallel with supervisorctl instead of sequentially

### ISSUE THREAD

**Issue by maxiwheat** [Supervisor/supervisor#1254]
When performing `supervisorctl stop all`, the commands seems to get stopped in parallel, i.e. all at the same time. In my case stopping all ~30 child processes takes 20 seconds to execute. The feedback supervisorctl gives me when I issue the command is that it waits for all processes to stop at the same time and then show all processes stopped at once.

When I issue `supervisorctl stop command1:* command2:* command3:* command4:* command5:* ` it takes a whopping 85 seconds to execute, and the feedback I get is that it waits for each command to stop before stopping the next one (stopping them sequentially).

Is it possible have the commands stopped the same way (in parallel) as `stop all` when passing them as argument to supervisorctl ? Is there a workaround for this ?

**Comment by mnaberez** [comment]
Duplicate of #723
